python/lancedb/query.py

```diff
@@ -198,7 +200,7 @@ def refine_factor(self, refine_factor: int) -> LanceQueryBuilder:
         self._refine_factor = refine_factor
         return self
 
-    def to_df(self) -> pd.DataFrame:
+    def to_df(self):
```

Comment:
It would be nice to keep this type hint.
```suggestion
    def to_df(self) -> "pd.DataFrame":
```